.github/workflows/test-asset.yml

```diff
@@ -9,14 +9,41 @@ on:
 
 jobs:
   check-docker-limit-before:
-   uses: terascope/workflows/.github/workflows/check-docker-limit.yml@f86f716e47d989b939d978befa7721c1f5b10134
+   uses: terascope/workflows/.github/workflows/check-docker-limit.yml@5307c8fb58de8d120cc025907cd417407f6c17cb
    secrets: inherit
 
   cache-docker-images:
     needs: check-docker-limit-before
-    uses: terascope/workflows/.github/workflows/cache-docker-images.yml@f86f716e47d989b939d978befa7721c1f5b10134
+    uses: terascope/workflows/.github/workflows/cache-docker-images.yml@5307c8fb58de8d120cc025907cd417407f6c17cb
     secrets: inherit
 
+  build-elasticsearch-assets:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # NOTE: Hard Coded Node Version array, should match array in build-and-publish-asset.yml
+        node-version: [18, 20, 22]
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+
+      - run: yarn && yarn setup
+
+      - run: yarn dlx teraslice-cli -v
+      - run: yarn dlx teraslice-cli assets build
```

Comment:
Is there any reason to leave the other `asset build`s elsewhere in here?

Reply:
No. I'll remove those now